core/src/main/scala/akka/persistence/cassandra/snapshot/CassandraSnapshotStore.scala

```diff
@@ -133,10 +133,10 @@ import akka.stream.alpakka.cassandra.scaladsl.{ CassandraSession, CassandraSessi
     case md +: mds =>
       load1Async(md)
         .map {
-          case DeserializedSnapshot(payload, OptionVal.Some(snapshotMeta)) =>
-            Some(SelectedSnapshot(md.withMetadata(snapshotMeta), payload))
           case DeserializedSnapshot(payload, OptionVal.None) =>
             Some(SelectedSnapshot(md, payload))
+          case DeserializedSnapshot(payload, some) =>
```

Comment:
Same here

Reply:
If the issue is very unlikely to occur, maybe best to wait to change. Another option is to have a config that controls whether you get old behaviour or new behaviour.

Reply:
I think a config is overkill. We should verify what the behaviour should be and if its different in my view it should go into 1.1.x